api/src/main/java/org/pmiops/workbench/initialcredits/InitialCreditsService.java

```diff
@@ -355,6 +355,15 @@ public void setInitialCreditsExpirationBypassed(DbUser user, boolean isBypassed)
     userInitialCreditsExpiration.setBypassed(isBypassed);
   }
 
+  public boolean isExpirationBypassed(DbUser user) {
+    DbUserInitialCreditsExpiration userInitialCreditsExpiration =
+        user.getUserInitialCreditsExpiration();
+
+    return userInitialCreditsExpiration != null
```

Comment:
can we add a comment explaining this is only for transition period, and mention the expected state after transition (would be nice to remove this after migration)